internal/vspd/update.go

```diff
@@ -254,7 +254,7 @@ func (v *Vspd) addToWallets(ctx context.Context, dcrdClient *rpc.DcrdRPC) {
 					}
 				}
 
-				v.log.Infof("Ticket added to voting wallet (wallet=%s, ticketHash=%s)",
+				v.log.Debugf("Ticket added to voting wallet (wallet=%s, ticketHash=%s)",
```

Comment:
I would think an admin would like to see this in the logs (without having to enable debug logging) in order to easily be able to determine if it was added properly when a user shows up later making claims they registered a ticket and it was accepted but didn't vote.

Reply:
Good point. The reason I downgraded this one is because it is logged for each wallet, i.e. 3+ times per ticket. I'm going to leave it at info but replace it with a single line.

Reply:
Meh, the commit message is wrong, but I plan to squash this down to a single commit before merging it anyway.

Reply:
The change to this looks good.  It's failing due to the move of pluralize though.  Resolving this bit.

Reply:
Yeah that was dumb. Reverted.